src/ui/Base/InitializationPlaceholder.ts

```diff
@@ -114,34 +115,66 @@ export class InitializationPlaceholder {
   public static NUMBER_OF_RESULTS = 10;
   public static NUMBER_OF_RESULTS_RECOMMENDATION = 5;
   public static INITIALIZATION_CLASS = 'coveo-during-initialization';
+  public static DEFAULT_OPTIONS: InitializationPlaceholderOption = {
+    waitForFirstQuery: false,
```

Comment:
I'm thinking maybe we don't even need an option for "waitForFirstQuery" ?

Could the InitializationPlaceholder *always* wait for the first query, it's just that in one instance it happens immediately (on the first automatic query).

Any chance we we could remove the "exception" and have only one code path that work correctly for both method of initialization ? (automatic query and non-automatic one).

Reply:
Good point, surely doable. I'll check what I can do.